table_foosball/database_test.rb

```diff
@@ -0,0 +1,79 @@
+require_relative './database'
+require_relative './table_foosball_01'
+require 'minitest/autorun'
+
+class DatabaseTest < Minitest::Test
+	def setup
```

Comment:
This means that down the line we can have your database as a completely separate project and potentially use it for other things.